3.5/runtime/windowsservercore-1903/Dockerfile

```diff
@@ -10,6 +10,14 @@ RUN curl -fSLo microsoft-windows-netfx3.zip https://dotnetbinaries.blob.core.win
     && del microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab `
     && powershell Remove-Item -Force -Recurse ${Env:TEMP}\*
 
+# Apply latest patch
```

Comment:
I'm not following why this is necessary.  Isn't this patch the OS LCU?  .NET for 1903 is patched separately from the OS.

Reply:
Ah, yep.